petri/src/vm/construct.rs

```diff
@@ -923,6 +923,12 @@ impl PetriVmConfigSetupCore<'_> {
             com2: true,
             vmbus_redirection: false,
             vtl2_settings: None, // Will be added at startup to allow tests to modify
+            vmgs_disk: Some(
+                disk_backend_resources::RamDiskHandle {
+                    len: vmgs_format::VMGS_DEFAULT_CAPACITY,
```

Comment:
fwiw, I'd be fine with having this hard-code some arbitrary petri-specific constant (i.e: I don't think it needs to remain in-sync with any other thing that integrates vmgs functionality)